shared/video-sdk/develop/real-time-transcription/project-implementation/macos.mdx

```diff
@@ -277,20 +277,20 @@ Take the following steps to start, stop and query a text-to-speech task in your
                 }
 
                 if let status = responseDict["status"] as? String, let taskId = responseDict["taskId"] as? String {
-                    self.speechToTextTaskId = taskId
+                    self.realTimeTranscriptionTaskId = taskId
 
                     if status == "IN_PROGRESS" {
-                        self.isSpeechToTextRunning = true
+                        self.isRealTimeTranscriptionRunning = true
                         DispatchQueue.main.async {
-                            self.speechToTextButton.title = "Stop speech to text"
+                            self.realTimeTranscriptionButton.title = "Stop real-time transcription"
```

Comment:
Is that purposefully not changed to "Real time transcription is running". 
```
self.showMessage(title: "Success", text: "Speech to text is running", delay: 0)
```
Multiple occurrence of the same at other places.